spec/support/acceptance/shared_examples.rb

```diff
@@ -46,3 +46,42 @@
     expect(num_incidents_after_puppet_run).to eql(num_incidents_before_puppet_run)
   end
 end
+
+# 'ictc' => 'incident creation test case'
+RSpec.shared_examples 'ictc' do |report_label: nil, noop_test: false|
```

Comment:
This might seem like "too much abstraction for tests", but it makes the resulting tests _a lot_ cleaner to read and maintain. Also, debugging failures isn't hard because the tests are simple enough that any issue will likely be in the setup (the `case` statements) instead of the actual test assertions.

In other words, this pattern is common and clear enough that I think the significantly reduced verbosity resulting from this shared example is worth keeping it.

Similar arg can be made for the corresponding `ictc` example used in the unit tests.